spec/fixtures/webmock-licenses-dir-with-multiple-license-files/dir.json

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
I was thinking we'd not have any webmock tests for this feature.

Reply:
Ah ok, I get you now. I'll instead inline these json files into `git_hub_project_spec.rb` in my next revision.